nb_train.py

```diff
@@ -4,87 +4,159 @@
 import nltk
 import random
 import pickle
-from sklearn.model_selection import train_test_split
 from nltk.corpus import stopwords
 from zipfile import ZipFile
 
-with ZipFile('imdb ratings.zip') as zf:
-    f = zf.open('movie.csv')
-    
-df = pd.read_csv(f)
-smalldf = df[:2000]
-smalldf = smalldf.drop_duplicates(ignore_index = True)
-smalldf.loc[smalldf['label'] == 0, 'label'] = 'neg'
-smalldf.loc[smalldf['label'] == 1, 'label'] = 'pos'
+DATA = 'movie.csv'
```

Comment:
`DATA` is a bit unclear; it could imply that the data is *literally* "movie.csv".

Instead let's use: `PATH_TO_DATA_FILE`

Reply:
The general rule for the names of constants is that they should be as informative as possible without being egregiously long